diagon-alley/src/Components/DetailedProduct/DetailedProduct.module.css

```diff
@@ -0,0 +1,100 @@
+.detailed_products {
+    margin: 6rem;
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+    justify-content: center;
+    column-gap: 1rem;
+    row-gap: 1rem;
+}
+
+.product_image {
+    background-color: hsla(0, 0%, 98%, 0.6);
+    width: 18rem;
+    height: 20rem;
+}
+
+.product_desc {
+    background-color: hsla(0, 0%, 98%, 0.7);
+    padding: 1rem;
+    width: 18rem;
+    display: flex;
+    flex-direction: column;
+    gap: 0.4rem;
```

Comment:
if it is less than 10px try to use pixel itself instead of rem.

Reply:
I have taken fontsize as 16px, that is 1rem = 16px, so took it this way